pipelines/h3c/idr2/v1/h3c_idr2_1.toml

```diff
@@ -127,18 +127,31 @@ args = ["{basename}", "${ABSCAL_OPTS:model_files_glob}", "${ABSCAL_OPTS:nInt_to_
         "${ABSCAL_OPTS:phs_conv_crit}", "${ABSCAL_OPTS:edge_cut}"]
 
 [XRFI]
```

Comment:
At some point before merging, I want to check with Josh that this won't break his current workflow.